<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>改变药品库存</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/back.jpg);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>药品名称：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>现有库存：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="NowStock"/>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>最小库存阈值设置：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="minStock"/>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>最大库存阈值设置：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="maxStock"/>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="comfirmButton">
     <property name="text">
      <string>确定修改</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="background_image.qrc"/>
 </resources>
 <connections/>
</ui>
